marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/flow-operator-role.json

```diff
@@ -4,7 +4,12 @@
   "role": [
     "rest-reader",
     "rest-extension-user",
-    "rest-writer"
+    "rest-writer",
```

Comment:
Thanks Srinath!

@aebadirad flow-operator-role has "rest-writer" today - does that it mean should have all the new *-writer roles as well - i.e. the ones inherited above by flow-developer-role?